### PR TITLE
MDATP/Requirements: add Linux Public Preview notes

### LINKED ISSUE #6430: Linux support is in preview
This page doesn't mention that the Linux support is only currently preview - not GA.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b8aca66a-5a8b-64d2-7201-a7b915c5d309
* Version Independent ID: c3cc0543-db20-a8f2-288c-d1dea5ea479f
* Content: [Minimum requirements for Microsoft Defender ATP - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/minimum-requirements)
* Content Source: [windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As noted in issue ticket #6430 (**Linux support is in preview**), this page does not have a description of Linux support for MD-ATP being Public Preview only for Linux at this time. This PR aims to rectify that information detail.

Thanks to @memildin (Mel) for reporting this issue.

**Changes proposed:**
- Add info comment behind "Linux" in the bullet point list
- Add a sentence at the end of the following Note blob
- Remove redundant phrase "that are" from a Note, to improve readability
- Add a missing plural S at the end of "Related topic" (2 links)
- Whitespace changes (might as well correct those)
    - add MarkDown indent marker compatibility spacing
    - remove redundant spacing in bullet point lists (only 1 needed)
    - remove redundant blank lines, 2 blank lines is sufficient spacing
    - remove redundant blanks at end-of-line (EOL whitespace)
    - replace a tab character with 4 blank spaces (GitHub codestyle)

**Ticket closure or reference:**

Closes #6430

### CODE REVIEW

**Comment by illfated** [comment]
@JohanFreelancer9 : Please perform a copy review on this content. I welcome suggestions for alternate phrasing and formatting for the added notes, in particular.

**Comment by illfated** [comment]
#sign-off

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ ms.topic: conceptual
 
 There are some minimum requirements for onboarding machines to the service. Learn about the licensing, hardware and software requirements, and other configuration settings to onboard devices to the service.
 
->Want to experience Microsoft Defender ATP? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-minreqs-abovefoldlink)
+> Want to experience Microsoft Defender ATP? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-minreqs-abovefoldlink)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:27]:
```suggestion
> Want to experience Microsoft Defender ATP? [Sign up for a free trial](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-minreqs-abovefoldlink).
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Microsoft Defender Advanced Threat Protection requires one of the following Micr
 - Microsoft 365 E5 Security
 - Microsoft 365 A5 (M365 A5)
 
-For detailed licensing information, see the [Product terms page](https://www.microsoft.com/en-us/licensing/product-licensing/products) and work with your account team to learn the detailed terms and conditions for the product. 
+For detailed licensing information, see the [Product terms page](https://www.microsoft.com/en-us/licensing/product-licensing/products) and work with your account team to learn the detailed terms and conditions for the product.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:43]:
```suggestion
For detailed licensing information, see the [Product terms page](https://www.microsoft.com/licensing/product-licensing/products) and work with your account team to learn the detailed terms and conditions for the product.
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 3]
```diff
@@ -82,24 +83,26 @@ Machines on your network must be running one of these editions.
...
-> Machines that are running mobile versions of Windows are not supported.
+> Machines running mobile versions of Windows are not supported.
 
 
 ### Other supported operating systems
--  macOSX
--  Linux
--  Android
+- macOSX
+- Linux (currently, MD-ATP is only available in Public Preview Edition for Linux)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:91]:
```suggestion
- Linux (currently, Microsoft Defender ATP is only available in the Public Preview Edition for Linux)
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 4]
```diff
@@ -82,24 +83,26 @@ Machines on your network must be running one of these editions.
...
+- Android
 
->[!NOTE]
->You'll need to know the exact Linux distros, Android, and macOS versions that are compatible with Microsoft Defender ATP for the integration to work. 
+> [!NOTE]
+> You'll need to know the exact Linux distros, Android, and macOS versions that are compatible with Microsoft Defender ATP for the integration to work.
+>
+> Also note that MD-ATP is currently only available in Public Preview Edition for Linux.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:97]:
```suggestion
> Also note that Microsoft Defender ATP is currently only available in the Public Preview Edition for Linux.
```

**Comment by illfated** [comment]
Thank you for the review. All suggestions have been committed.

I presume we may ask for a final review now.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**